packages/rollup-plugin-html/src/output/emitAssets.ts

```diff
@@ -57,7 +82,54 @@ export async function emitAssets(
 
       let ref: string;
       let basename = path.basename(asset.filePath);
+      const emittedExternalAssets = new Map();
       if (asset.hashed) {
+        if (basename.endsWith('.css') && options.bundleAssetsFromCss) {
+          let updatedCssSource = false;
+          const { code } = await transform({
+            filename: basename,
+            code: asset.content,
+            minify: false,
+            visitor: {
+              Url: url => {
+                // Support foo.svg#bar
+                const [filePath, idRef] = url.url.split('#');
+
+                if (shouldHandleAsset(url.url)) {
```

Comment:
can you pass `filePath` instead of `url` and simplify the extension check for the SVG?

Reply:
fixed